tensorflow/c/kernels.cc

```diff
@@ -357,6 +357,10 @@ void TF_GetInput(TF_OpKernelContext* ctx, int i, TF_Tensor** tensor,
     return;
   }
   const ::tensorflow::Tensor& cc_tensor(cc_ctx->input(i));
+  if ((&cc_tensor) == nullptr) {
```

Comment:
Thanks for the fix! 

I have a quick question because I'm a little confused how `&cc_tensor` can be `nullptr`, same with `cc_ctx->input(i)` since it also returns a reference. Based on the prior code the only thing that looks like it might be nullptr is `cc_ctx` which is cast from `ctx` - is the bug resolved if we check for `(ctx == nullptr)`?

Reply:
In [cc_ctx->input(i)](https://github.com/tensorflow/tensorflow/blob/v2.12.0/tensorflow/core/framework/op_kernel.cc#L436) , the return tensor will be a reference of nullptr if params_->inputs[index].tensor is a nullptr.  So the cc_tensor may also a reference of nullptr. Following simple case can reproduce this phenomenon.
`
    using T = int;
    T* src = nullptr;
    T& dst = *src;
    if(&dst==nullptr) std::cout << "dst is nullptr" << std::endl;
`